src/app/common/services/comment-service.coffee

```diff
@@ -0,0 +1,37 @@
+angular.module("doubtfire.common.services.comments", [])
+#
+# Services for transferring comment resources
+#
+.factory("CommentResourceService", ->
```

Comment:
Refactor - remove globals used to pass task and comment related urls around.

Add comment related URLs into the Comment itself
Tasks should be available
Pass via scope if needed

Keep the support for context and checking capability/support from browser